kmesh_compile_env_pre.sh

```diff
@@ -14,11 +14,11 @@ function install_libboundscheck() {
 function dependency_pkg_install() {
     if command -v apt > /dev/null; then
 	    # apt install 
-	    apt-get update && apt-get install -y git make clang libbpf-dev llvm linux-tools-generic protobuf-compiler libprotobuf-dev libprotobuf-c-dev protobuf-c-compiler cmake golang
+	    apt-get update && apt-get install -y git make clang libbpf-dev llvm linux-tools-generic protobuf-compiler libprotobuf-dev libprotobuf-c-dev protobuf-c-compiler cmake pkg-config
```

Comment:
why add `pkg-config`

Reply:
pkg-config is unavailable on some Ubuntu distros that we have met.